pom.xml

```diff
@@ -98,6 +98,26 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
+                <executions>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>11</version>
```

Comment:
Any reason for the requirement?

It's still compiling against JDK 8 APIs right?

Reply:
It produces an MR JAR for at least one module, so you need to build with an appropriate JDK to support all the layers.  The base layer is Java 8 though.